common/src/main/java/com/wynntils/functions/generic/ConditionalFunctions.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.functions.generic;
+
+import com.wynntils.core.functions.GenericFunction;
+import com.wynntils.core.functions.arguments.FunctionArguments;
+import java.util.List;
+
+public class ConditionalFunctions {
+    public static class IfStringFunction extends GenericFunction<Object> {
+        @Override
+        public Object getValue(FunctionArguments arguments) {
+            if (arguments.getArgument("condition").getBooleanValue()) {
+                return arguments.getArgument("ifTrue").getValue();
+            } else {
+                return arguments.getArgument("ifFalse").getValue();
+            }
+        }
+
+        @Override
+        public FunctionArguments.RequiredArgumentBuilder getRequiredArgumentsBuilder() {
+            return new FunctionArguments.RequiredArgumentBuilder(List.of(
+                    new FunctionArguments.Argument<>("condition", Boolean.class, null),
+                    new FunctionArguments.Argument<>("ifTrue", String.class, null),
+                    new FunctionArguments.Argument<>("ifFalse", String.class, null)));
+        }
+
+        @Override
+        public List<String> getAliases() {
+            return List.of("if_str");
+        }
+    }
```

Comment:
@magicus I know this is not what you imagined. But I can't use `T.class` and I need that in `getRequiredArgumentsBuilder`. So I resorted to this smaller base class. I hope it is fine.

Reply:
Yeha, okay. I will not hold limitation in Java's type system against you. :)